GiEnJul/ClientApp/src/components/admin/common/GiverTable.tsx

```diff
@@ -37,18 +41,21 @@ const Datatable: React.FC<Props> = ({ data }) => {
     }
   };
 
-  const handleMatched = (input: Boolean) => {
-    if (input) {
-      return classes.matched;
-    } else {
-      return classes.notMatched;
-    }
-  };
-
   return (
     <Container>
       {data.map((giver, index) => (
-        <Accordion key={`acc_giver_${index}`}>
+        <Accordion
+          onChange={() => handleGiverChange(giver.rowKey, giver.partitionKey)}
+          key={`acc_giver_${index}`}
+          style={
+            giver.isSelected
+              ? {
+                  background:
+                    "linear-gradient(45deg, #D6F0EB 30%, #E2FFF9 90%)",
```

Comment:
Do we want styling directly in our components?

Reply:
Written comments in the code so that we change this when implementing styling